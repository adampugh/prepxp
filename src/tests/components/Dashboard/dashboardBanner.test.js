import React from "react";
import { shallow } from "enzyme";

import DashboardBanner from "../../../components/Dashboard/dashboardBanner";

describe("<DashboardBanner />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<DashboardBanner />);
        expect(wrapper).toMatchSnapshot();
    });
});