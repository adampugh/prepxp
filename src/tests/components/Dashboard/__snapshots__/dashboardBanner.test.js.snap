// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardBanner /> should match snapshot 1`] = `
<section
  className="dashboard__banner"
>
  <div
    className="container"
  >
    <p>
      Create a new list of interview questions
    </p>
    <button
      className="btn"
    >
      Create
    </button>
  </div>
</section>
`;
